module-4/backend/week_one.md

```diff
@@ -1,19 +1,50 @@
 ## Week One - Module 4 Recap
 
-Fork this respository. Answer the questions to the best of your ability. Try to answer them with limited amount of external research. These questions cover the majority of what we've learned this week. 
+1. What's the most useful thing you learned from completing the intermission week work?
 
-Note: When you're done, submit a PR. 
+  The most important thing that I learned from completing the intermission week work is JavaScript syntax.
 
-1. What's the most useful thing you learned from completing the intermission week work?
 2. What are some tools to help debug JavaScript code?
+
+  Some tools to help me debug JavaScript are pry, console.log, and the console in your browser.
+
 3. What are some tools you need in order to unit test your JavaScript?
+
+  You need mocha and chai to test your JavaScript.
+
 4. What is the syntax for invoking a function?
+
+  function name(){
+
+  }
+
+  name()
+
 5. What's `this` in JavaScript?
+
+  `this` is the outer object of the current object that you are in.
+
 6. What is Webpack and why is it useful?
+
+  Webpack is a bundler or boilerplate for JavaScript to help you build a file blueprint faster.
```

Comment:
It is not only used to build a file blueprint, you can use transpilers such as babel to compile ES6 code to ES5, you can build all JS files into 1 file, as well as minifying this file so that its smaller and faster for browsers